src/supplemental/nanolib/conf_ver2.c

```diff
@@ -1413,6 +1414,7 @@ conf_exchange_parse_ver2(conf *config, cJSON *jso)
 		node->topic              = NULL;
 		node->name               = NULL;
 		node->rbufs              = NULL;
+		node->streamType         = 0;
```

Comment:
it would be better to rename to stream_type

Reply:
yup